README.md

```diff
@@ -49,6 +49,16 @@ For help and more options, type:
 page-computed-style -h
 ```
 
+## Installing & Using Locally
+
+If you would rather installing the package locally, omit `-g` parameter and the usage will be directly from `node_modules/page-computed-style/index.js`:
```

Comment:
I'm not a native english speaker, but I think it should be "If you would rather install" instead of "If you would rather installing"

Reply:
Neither am I. You are right, it should be "... rather install".